src/components/transactions/transactions.component.js

```diff
@@ -15,6 +15,7 @@
  */
 import AppTransactions from './transactions.module';
 import template from './transactions.html';
+import { identity } from 'ui-router-core';
```

Comment:
It's probably added automatically, not needed and fails on ESlint (check the Jenkins report)